### PR TITLE
MONA-24 - testing key value list component

### PR DESCRIPTION
**Pull request by mstone6769**

### CODE REVIEW

File: lib/dist/iui-lists.js [hunk 1]
```diff
@@ -9,15 +9,33 @@
 (function() {
   'use strict';
 
-  angular.module('iui.keyValue', [])
-    .directive('iuiKeyValue', list);
+  angular.module('iui.keyValue', []);
```
Comment (acsearles) [inline lib/dist/iui-lists.js]: what is this magic you have here where this is no longer a directive? This part confuses me.

Comment (acsearles) [inline lib/dist/iui-lists.js]: oh nevermind. The directive is below.

File: lib/src/iui-list/_iui-list.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+.list-group-item-link {
+  @extend a.list-group-item;
+  padding: 10px 15px;
+  display: block;
+  padding-right: 30px;
+  position: relative;
+  &:before {
+    font-family: 'Glyphicons Halflings';
```
Comment (acsearles) [inline lib/src/iui-list/_iui-list.scss]: Is this a new font we're using for Font Icons? I thought we were using the Bootstrap ones.

Comment (mcastre) [inline lib/src/iui-list/_iui-list.scss]: Glyphicons are the default Bootstrap font icons

Comment (acsearles) [inline lib/src/iui-list/_iui-list.scss]: Yeah ignore that. I didn't realize Halflings was apart of the name. That part threw me off. Thanks.

Comment (mstone6769) [inline lib/src/iui-list/_iui-list.scss]: I believe Glyphicons Haflings is the name of the Bootstrap icon font set
http://getbootstrap.com/components/#glyphicons

**Pull request merged**